workspace/__lib__/xod/bitmaps/16x16-worried-face-apple/patch.cpp

```diff
@@ -0,0 +1,37 @@
+
+// clang-format off
+{{#global}}
+static const unsigned char apple_worried_face_16x16[] PROGMEM = {
+
+0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x10,0x60,0x31,0x00,0x31,0x00,0x10,0x60,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00
+,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x49,0x60,0xbc,0x62,0xfe,0x8a,0xff,0x4f,0xff,0x4f,0xfe,0xab,0xc4,0xc3,0x51,0xc0,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00
+,0x00,0x00,0x00,0x00,0x00,0x00,0x9b,0x20,0xfe,0xca,0xff,0xd6,0xff,0xf9,0xff,0xfa,0xff,0xfa,0xff,0xf9,0xff,0xd7,0xfe,0xec,0xab,0xa1,0x08,0x20,0x00,0x00,0x00,0x00
+,0x00,0x00,0x00,0x00,0x92,0xe0,0xfe,0xcb,0xff,0xb3,0xff,0xb3,0xff,0xd5,0xff,0xd6,0xff,0xd6,0xff,0xd5,0xff,0xb3,0xff,0xb3,0xff,0x0d,0xb3,0x81,0x00,0x00,0x00,0x00
+,0x00,0x00,0x41,0x40,0xfd,0xe5,0xcd,0x49,0xa4,0x05,0xb4,0xc8,0xff,0xb0,0xff,0xd0,0xff,0xd0,0xff,0xd0,0xbc,0xe8,0xa4,0x05,0xc5,0x08,0xfe,0x27,0x61,0xc0,0x00,0x00
+,0x00,0x00,0xbb,0x60,0xfe,0x28,0xdd,0xe8,0xff,0x6a,0xff,0x8b,0xff,0xab,0xff,0xac,0xff,0xab,0xff,0xab,0xff,0x8b,0xff,0x4a,0xe6,0x29,0xf6,0x48,0xd4,0x01,0x00,0x00
+,0x10,0x40,0xec,0xa1,0xfe,0x26,0xfe,0xa7,0xfe,0xe7,0xf6,0xc7,0xff,0x48,0xff,0x68,0xff,0x68,0xff,0x48,0xf6,0xe7,0xfe,0xe7,0xfe,0xa7,0xfe,0x46,0xf5,0x02,0x20,0xa0
+,0x28,0xa0,0xf4,0xc1,0xfd,0xe4,0xfe,0x65,0xbc,0x43,0x92,0xc1,0xfe,0xe6,0xff,0x06,0xff,0x06,0xff,0x06,0x93,0x01,0xbc,0x22,0xfe,0x65,0xfe,0x05,0xf5,0x02,0x41,0x20
+,0x28,0xa0,0xec,0xa1,0xfd,0xc4,0xfe,0x25,0xd5,0x24,0xb4,0x23,0xfe,0xc6,0xfe,0xc6,0xfe,0xc6,0xfe,0xc6,0xbc,0x23,0xd5,0x04,0xfe,0x25,0xfd,0xc4,0xf4,0xc1,0x41,0x20
+,0x10,0x40,0xdc,0x20,0xfd,0x83,0xfe,0x06,0xfe,0x46,0xfe,0x66,0xfe,0x86,0xfe,0xa6,0xfe,0xa6,0xfe,0x86,0xfe,0x86,0xfe,0x46,0xfe,0x06,0xfd,0x83,0xe4,0x61,0x20,0xa0
+,0x00,0x00,0xab,0x00,0xf5,0x02,0xfd,0xa4,0xfe,0x06,0xfe,0x26,0xd4,0xe4,0xab,0xa2,0xab,0xa2,0xd4,0xe4,0xfe,0x26,0xfe,0x06,0xfd,0xc4,0xf5,0x22,0xc3,0x80,0x00,0x00
+,0x00,0x00,0x41,0x20,0xdc,0x41,0xf5,0x42,0xfd,0xa4,0xc4,0x23,0xa3,0x42,0xbc,0x43,0xbc,0x43,0xa3,0x62,0xc4,0x23,0xfd,0xa4,0xfd,0x42,0xe4,0x81,0x61,0xa0,0x00,0x00
+,0x00,0x00,0x00,0x00,0x92,0x80,0xe4,0x81,0xf5,0x22,0xed,0x03,0xfd,0x84,0xfd,0x44,0xfd,0x44,0xfd,0x84,0xe5,0x03,0xf5,0x22,0xe4,0xa1,0xa2,0xe0,0x00,0x00,0x00,0x00
+,0x00,0x00,0x00,0x00,0x08,0x20,0x92,0x80,0xdc,0x41,0xec,0xc2,0xf4,0xc2,0xf4,0xe2,0xf5,0x02,0xf4,0xe2,0xec,0xc2,0xe4,0x61,0xa2,0xe0,0x10,0x40,0x00,0x00,0x00,0x00
+,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x51,0x60,0xb3,0x20,0xdc,0x21,0xe4,0x61,0xe4,0x61,0xdc,0x21,0xbb,0x60,0x61,0xa0,0x08,0x20,0x00,0x00,0x00,0x00,0x00,0x00
+,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x20,0x80,0x39,0x00,0x41,0x00,0x28,0xa0,0x00,0x20,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00
+
+};
+{{/global}}
+// clang-format on
+
+struct State {};
+
+{{ GENERATED_CODE }}
+
+void evaluate(Context ctx) {
+    ValueType<output_BMP>::T bmp;
+    bmp.bitmapBuffer = apple_worried_face_16x16;
```

Comment:
It’s OK to hold the images this way. However, I see no point in making the definition `{{#global}}`. Keep it scoped and use a simple potentially non-unique name.